### DISCUSSION TITLE
Py object show.

### DISCUSSION THREAD

**Discussion by J-C-Q** [cjdoris/PythonCall.jl#503] [Q&A]
Py objects are shown in the REPL with a 'Python:' prefix. How can I disable this?

**Comment by cjdoris** [comment]
There's no option to globally disable it.

**Reply by J-C-Q** [reply]
I tried to add it. [Added global option to disable 'Python:' prefix in show #505](https://github.com/JuliaPy/PythonCall.jl/pull/505)
